README.md

```diff
@@ -4,13 +4,29 @@
 
 2. `console.log` the length of a string.
 
-3. `console.log` the last character of any string. 
+3. `console.log` the last character of any string.
 
 4. Create a code block that takes a single string variable, and returns a copy of the string with the last letter capitalized. For example:
 
 ```js
 'jimmy'
 // => 'jimmY'
+
+//Answers to 1-3 Lines 16-19
+let str = "pearl"
+console.log(str[0]);
+console.log(str.length);
+console.log(str.charAt(str.length-1));
+
+//Answer to 4 lines 22-29
+let str = "pearl"
+let lastCap = str.slice(0,4)+ str[4].toUpperCase()
```

Comment:
What would happen if we change str to "Alejandro"? Would it still give us "AlejandrO"? This is example of hard coding which is not a good practice. Aways try to write a code that can work for differing inputs.